bundle/bin/rke2-uninstall.sh

```diff
@@ -30,13 +34,21 @@ uninstall_disable_services()
 
 uninstall_remove_files()
 {
+
     find "${INSTALL_RKE2_ROOT}/lib/systemd/system" -name rke2-*.service -type f -delete
     rm -f "${INSTALL_RKE2_ROOT}/bin/rke2"
```

Comment:
These are all managed by the RPM, I'm not sure we should be out-of-band removing files... If we are, then we should probably do the `yum -y remove rke2-*` before we actually go through and do the `rm -f ` of things

Reply:
(This is in reference to the detection up above of the `/usr` vs `/usr/local` for `INSTALL_RKE2_ROOT`)